portfolio/src/main/webapp/style.css

```diff
@@ -6,4 +6,18 @@
 
 #greeting-container {
   margin-top: 20px;
+  color: orange;
+  font-size: large;
+  font-weight: bold;
+}
+
+img {
```

Comment:
Same. Tag name shouldn't be used for selection in CSS. Use a class name and add the class to image tag.